docs/source/examples/substrafl/go_further/run_iris_sklearn.ipynb

```diff
@@ -0,0 +1,679 @@
+{
```

Comment:
<p>Ok, the missing coma is probably on all notebooks, given that this paragraph has been copy-pasted everywhere</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Substra/substra-documentation/pull/368/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>